assets/css/team.css

```diff
@@ -14,6 +14,104 @@ body{
 	width: auto;
 }
 
+/* contact form */
+
+#contact .form-control{
```

Comment:
All changes to be done in a contact.css

Reply:
Before team page is contact page... after getting some changes it changed by own...